api/src/policies/users-policy.ts

```diff
@@ -32,15 +32,18 @@ export class UsersPolicy extends BasePolicy<User> {
   }
 
   permittedAttributes(): Path[] {
-    return [
-      "email",
-      "firstName",
-      "lastName",
-      "position",
-      {
-        groupMembershipAttributes: ["departmentId", "divisionId", "branchId", "unitId"],
-      },
-    ]
+    if (this.user.roleTypes.includes(RoleTypes.SYSTEM_ADMIN)) {
+      return [
+        "email",
+        "firstName",
+        "lastName",
+        "position",
+        {
+          groupMembershipAttributes: ["departmentId", "divisionId", "branchId", "unitId"],
+        },
+      ]
+    }
+    return ["email", "firstName", "lastName"]
```

Comment:
Generally I like adding a space under a "guard clause", unless using multiple guard causes.
```suggestion

    return ["email", "firstName", "lastName"]
```